packages/tooltip/src/Tooltip.spec.tsx

```diff
@@ -435,7 +435,8 @@ describe('packages/tooltip', () => {
     });
 
     expect(
-      getByTestId(tooltipTestId).parentElement?.parentElement?.className,
+      getByTestId(tooltipTestId).parentElement?.parentElement?.parentElement
+        ?.className,
     ).toBe('test-classname');
```

Comment:
we can probably make this test more robust doing something like
```suggestion
    const matchedElements = document.querySelectorAll('body > .test-classname');
    expect(matchedElements).toHaveLength(1);
    const portalRoot = matchedElements.item(0)!;

    expect(portalRoot).toContainElement(getByTestId(tooltipTestId));
```

Reply:
Hmm... Usually not a fan of this approach, but in this case I kinda like it. I'll give that a shot.

Reply:
Done 👍